src/utilities/ivao.service.ts

```diff
@@ -2,37 +2,127 @@ import { HttpService, Injectable, Logger } from '@nestjs/common';
 import { map, tap } from 'rxjs/operators';
 import { CacheService } from '../cache/cache.service';
 
-// TODO: investigate
-// For some reason iconv is not working with import
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const iconv = require('iconv-lite');
+type Client = {
```

Comment:
**suggestion:** be consistent and use `interface` all throughout the file

Reply:
I agree, I'll update this soon.